metadata-ws/src/test/resources/features/release-date.feature

```diff
@@ -0,0 +1,124 @@
+Feature: Transitive release date control for child objects of a Study
+
+  Scenario Outline: Objects to which at least released study links should be accessible, and vice versa
+
+    # Create the common taxonomy
+    When I request POST taxonomies with 9606 for ID, Homo Sapiens for name and NONE for ancestors
+    Then set the URL to TAXONOMY
+    And the response code should be 201
+
+    # Create the common reference sequences
+    Given I request POST /reference-sequences with JSON-like payload:
+    """
+          "name": "GRCh37",
+          "patch": "p2",
+          "accessions": ["GCA_000001407.3", "GCF_000001407.14"],
+          "type": "ASSEMBLY",
+          "taxonomy": "TAXONOMY"
+    """
+    Then set the URL to REFERENCE_SEQUENCE
+    And the response code should be 201
+
+    # Create two studies with the release date of "yesterday"
```

Comment:
This block should be the following succession:

1. When
2. And
3. Then
4. When
5. And
6. Then

This template can be used as reference for others below.

Reply:
Fixed in https://github.com/EBIvariation/metadata-ws/pull/53/commits/2da2671086076bae4c89bb2b6f79b4363089ef75.